modules/tinymce/src/plugins/table/main/ts/selection/SelectionTargets.ts

```diff
@@ -134,7 +155,17 @@ export const getSelectionTargets = (editor: Editor, selections: Selections): Sel
     onSetup(api, (targets) => isCaption(targets.element) || getClipboardData().isNone() || isDisabledFromLocked(lockedDisable));
   const onSetupMergeable = (api: UiApi) => onSetup(api, (_targets) => isDisabledForSelection('mergeable'));
   const onSetupUnmergeable = (api: UiApi) => onSetup(api, (_targets) => isDisabledForSelection('unmergeable'));
-
+  const onSetupTableWithCaption = (api: Toolbar.ToolbarToggleButtonInstanceApi) => {
+    return onSetupWithToggle(api, Fun.never, (targets) => {
+      const potentialTables = TableLookup.table(targets.element, Util.getIsRoot(editor));
```

Comment:
This will only ever return a single table, so this shouldn't be a plural. Additionally, the filter is not needed below, so please remove that as well.